packages/animation/src/components/test/WAAPIWrapper.js

```diff
@@ -78,13 +78,13 @@ describe('StoryAnimation.WAAPIWrapper', () => {
       )
     );
     const ElementsWithWrapper = ({
-      // eslint-disable-next-line react/prop-types
+      // eslint-disable-next-line react/prop-types -- @todo
```

Comment:
Let's just disable the `react/prop-types` rule completely for test files, doesn't really add value there.

Can do that here I think:

https://github.com/GoogleForCreators/web-stories-wp/blob/700032264a04185f9ca2b5232e3fe61a8ca34313/.eslintrc#L273-L313